### PR TITLE
Add new flow directions

### PR DESCRIPTION
**Pull request by Dev-Zhao**

### CODE REVIEW

File: Blish HUD/Controls/FlowPanel.cs [hunk 1]
```diff
@@ -152,19 +200,50 @@ public class FlowPanel : Panel {
...
+                if (outerPadX > lastLeft - child.Width)
+                {
+                    currentBottom = nextBottom + _controlPadding.Y;
+                    lastLeft = this.Width - outerPadX;
+                }
+
+                child.Location = new Point((int)(lastLeft - child.Width), (int)currentBottom);
+
+                lastLeft = child.Left - _controlPadding.X;
+                Debug.WriteLine(lastLeft - child.Width);
```
Comment (greaka) [inline Blish HUD/Controls/FlowPanel.cs]: You probably forgot this Debug statement in here

**Review by greaka — CHANGES_REQUESTED** [review]
Looks good!

Can you please revert the formatting changes? They were probably automatically introduced by your IDE

File: Blish HUD/Controls/FlowPanel.cs (hunk 1 above)
Comment (Dev-Zhao) [reply Blish HUD/Controls/FlowPanel.cs]: I have fixed the formatting and removed the Debug statement.

**Review by greaka — APPROVED** [review]

**Comment by Dev-Zhao** [comment]
Please ignore this pull request. I accidentally committed changes that are in #382.
I have submitted a new pull request - #383

**Pull request closed without merging**